src/webpack/optimization.ts

```diff
@@ -4,18 +4,18 @@ import UglifyJsPlugin from 'uglifyjs-webpack-plugin'
  * Webpack optimization
  * @type {function} optimization
  */
-const optimization = bud => ({
+const optimization = (bud: Bud) => ({
```

Comment:
Todo: related to the other todo about mapping Builder outputs to webpack's `Configuration` types definitions -- the methods themselves should get those same types/interfaces set on them explicitly. This will be helpful determining if problems are happening in the `BuilderController` or within a `Builder` method.

Reply:
done